webrtc-stats.html

```diff
@@ -1569,8 +1570,20 @@ <h2>
                   Transport Independent Application Specific (TIAS) bitrate [[!RFC3890]].
                   Typically, the target bitrate is a configuration parameter provided to the
                   codec's encoder and does not count the size of the IP or other transport layers
-                  like TCP or UDP. It is measured in bits per second and the bitrate is calculated
-                  over a 1 second window.
+                  like TCP or UDP. It is measured in bits per second. This statistic gives the
+                  configured target bitrate at the time of the stats timestamp.
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>totalTargetBytesEncoded</code></dfn> of type <span class=
```

Comment:
totalEncodedBytesTarget?

Reply:
do we need Frames?  because the bytes belong to a target frame size.
do we need total? although we have not used total for other metrics. Perhaps in this case we do need it.